src/host/main.cpp

```diff
@@ -358,26 +368,55 @@ int main(int argc, char** argv)
     public_rpc_address = rpc_address;
   }
 
-  if (domain.empty() && !ds::is_valid_ip(rpc_address.hostname.c_str()))
+  uint32_t oe_flags = 0;
```

Comment:
Is there an OE type for this?

Reply:
It doesn't look like it: https://github.com/openenclave/openenclave/search?p=1&q=OE_ENCLAVE_FLAG_DEBUG&unscoped_q=OE_ENCLAVE_FLAG_DEBUG